go/cli/internal/cli/install.go

```diff
@@ -25,28 +53,28 @@ func InstallCmd(ctx context.Context, c *ishell.Context) {
 		return
 	}
 
-	args := []string{
-		"upgrade",
-		"--install",
-		"kagent",
-		"oci://ghcr.io/kagent-dev/kagent/helm/kagent",
-		"--version",
-		Version,
-		"--namespace",
-		cfg.Namespace,
-		"--create-namespace",
-		"--wait",
-		"--set",
-		"openai.apiKey=" + os.Getenv("OPENAI_API_KEY"),
-	}
-	cmd := exec.CommandContext(ctx, "helm", args...)
 	s := spinner.New(spinner.CharSets[35], 100*time.Millisecond)
-	s.Suffix = " Installing kagent"
+
+	// First install kagent-crds
+	s.Suffix = " Installing kagent-crds"
 	s.Start()
+	if output, err := installChart(ctx, "kagent-crds", cfg.Namespace, Version, nil, s); err != nil {
+		// Check for various CRD existence scenarios, this is to be compatible with
+		// original kagent installation that had CRDs installed together with the kagent chart
+		if strings.Contains(output, "exists and cannot be imported into the current release") {
+			s.Stop()
+			c.Println("Warning: CRDs already exist but not managed by helm, you might need to delete them manually to make them fully managed by helm.")
+			s.Start()
+		} else {
+			c.Println("Error installing kagent-crds:", output)
+			return
+		}
+	}
```

Comment:
Not a requirement, but since we are in a `repl` anyway we could offer to do this for the user.

Reply:
That is a good point. 

Here is what I am thinking:
1. InstallCmd - still succeeds (to avoid breaking change) when CRDs are not managed by helm, but warns that users need to manually delete CRDs or run uninstall cmd to delete CRDs.
2. UninstallCmd - adding steps in uninstall to delete CRDs (by kubctl?) if helm release not found but CRDs are there.

how do you think?